tox.ini

```diff
@@ -60,6 +60,10 @@ passenv =
 deps =
   -r {env:BUNDLE_TEST_PATH}/requirements.txt
 
+[testenv:uats]
+allowlist_externals = tox
+commands =
+    tox -c uat -- --model kubeflow {posargs}
```

Comment:
for now, this is great.  But can you take note that we try and clean up this tox file once we know exactly how everything fits together?  Our CI will need to execute tests (deploy + uats) for each supported bundle.  It can work now, but doesn't feel like a cohesive story yet in the tox.ini

Reply:
Yep, this is obviously just the temporary way to integrate this into the repo.